.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+Metrics/LineLength:
+  Max: 120
+Style/Documentation:
+  Enabled: false
+
+AllCops:
+  TargetRubyVersion: 2.2.1
+  Exclude:
+    - 'db/migrate/*'
+
+Rails:
+  Enabled: true 
+
+Style/Encoding:
+  EnforcedStyle: when_needed
+
+Style/IndentationConsistency:
+	EnforcedStyle: rails
```

Comment:
Oooor, wait until I run rubocop -a on it, and it will change it to 2 spaces anyway. Bam.